hazelcast/src/main/java/com/hazelcast/config/DataConnectionConfig.java

```diff
@@ -181,6 +184,26 @@ public DataConnectionConfig setProperties(Properties properties) {
         return this;
     }
 
+    /**
```

Comment:
not caused by this PR: `setProperty` is not overridden in `DataConnectionConfigReadOnly`

Reply:
Nice catch!